.claude/commands/handle-pr-reviews.md

```
@@ -0,0 +1,102 @@
+現在のブランチのPRに対して実行:
+```
+/handle-pr-reviews
+```
+
+特定のPR番号を指定して実行:
+```
+/handle-pr-reviews 53
+```
+
+実装時の$ARGUMENTS使用例:
+```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)